Issue when installing Jupyter Lab extension

When doing:

```
jupyter labextension install js
```

I get the following:

```
ModuleNotFoundError: Module not found: Error: Can't resolve '../shaders/image-fragment.glsl' in '/home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/lib'
    at factory.create (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/webpack/lib/Compilation.js:535:10)
    at factory (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:397:22)
    at resolver (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
    at asyncLib.parallel (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
    at /home/tom/python/dev/share/jupyter/lab/staging/node_modules/neo-async/async.js:2825:7
    at /home/tom/python/dev/share/jupyter/lab/staging/node_modules/neo-async/async.js:6886:13
    at normalResolver.resolve (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
    at doResolve (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
    at hook.callAsync (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at resolver.doResolve (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:37:5)
    at hook.callAsync (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at hook.callAsync (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:27:1)
    at resolver.doResolve (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
    at hook.callAsync (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn42 (eval at create (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
    at hook.callAsync (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:27:1)
    at resolver.doResolve (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
    at hook.callAsync (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn1 (eval at create (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
    at hook.callAsync (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
    at _fn0 (eval at create (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at fs.stat (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/DirectoryExistsPlugin.js:22:13)
    at process.nextTick (/home/tom/python/dev/share/jupyter/lab/staging/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:73:15)
    at process._tickCallback (internal/process/next_tick.js:61:11)
resolve '../shaders/image-fragment.glsl' in '/home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/lib'
  using description file: /home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/package.json (relative path: ./lib)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/package.json (relative path: ./shaders/image-fragment.glsl)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/shaders/image-fragment.glsl doesn't exist
      .wasm
        Field 'browser' doesn't contain a valid alias configuration
        /home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/shaders/image-fragment.glsl.wasm doesn't exist
      .mjs
        Field 'browser' doesn't contain a valid alias configuration
        /home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/shaders/image-fragment.glsl.mjs doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/shaders/image-fragment.glsl.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/shaders/image-fragment.glsl.json doesn't exist
      as directory
        /home/tom/python/dev/share/jupyter/lab/staging/node_modules/jupyter-astroimage/shaders/image-fragment.glsl doesn't exist
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/js/package.json
+++ b/js/package.json
@@ -17,7 +17,8 @@
   ],
   "files": [
     "lib/**/*.js",
-    "dist/*.js"
+    "dist/*.js",
+    "shaders/*"
   ],
   "scripts": {
     "clean": "rimraf dist/",

